torch/distributed/_sharded_tensor/api.py

```diff
@@ -615,3 +633,26 @@ def __setstate__(self, state):
                 f'but at load time was {global_world_size}')
 
         self._post_init()
+
+
+def create_tensor_from_params(*size, local_device, tensor_init_params: TensorInitParams):
```

Comment:
nit: make this `_create_tensor_from_params` since it is an internal method and isn't meant to be a public API.

Reply:
Oh yes, fixed.